terraform/modules/fail-over-cname/main.tf

```diff
@@ -12,7 +12,7 @@ resource "cloudflare_record" "public-cname" {
   comment = "For easy fail over. ${local.primary_region_summary} / ${local.secondary_region_summary}"
   name    = "${var.subdomain}.${var.cloudflare_zone_name}"
   proxied = true
-  ttl     = 60 # seconds
+  ttl     = 1 # ttl must be set to 1 when proxied is true
```

Comment:
This is fine, and the comment is helpful. In other places, we simply omit the `ttl` parameter.

Reply:
I looked at the API documentation to get more background. It says that 1 means "auto".

Reply:
Ahh, thanks. That's good to know. 👍